src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1779,6 +1793,17 @@ static void ConsoleSettings()
 		mq::imgui::HelpMarker("Set the number of lines to keep in the scrollback buffer. Any lines above this amount will be deleted from the top of the buffer and won't be available for viewing in the console. Larger numbers here may cause performance issues like hitching or FPS slowdowns.");
 
 		ImGui::NewLine();
+
+		if (ImGui::SliderInt("Main Console Font Size", &s_consoleFontSize, 10, 30))
```

Comment:
Changing fonts is a pretty heavy operation, this shouldn't be a slider because you want to avoid all the intermediate set operations. Something like a drop-down with explicit supported values would be much better